venues/auai.org/UAI/2018/python/get-pdfs.py

```diff
@@ -0,0 +1,29 @@
+import requests
+import os
+import openreview
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+print 'connecting to {0}'.format(client.baseurl)
+
+submission_invitation = 'auai.org/UAI/2018/-/Blind_Submission'
+submissions = client.get_notes(invitation=submission_invitation)
+
+if not os.path.exists('../pdfs'):
+    os.makedirs('../pdfs')
+
+for n in submissions:
+    pdf_url = 'https://openreview.net{0}'.format(n.content['pdf'])
```

Comment:
Shouldn't this be client.baseurl instead of https://openreview.net?

Reply:
yes, thanks pam!